sanic/static.py

```diff
@@ -95,10 +96,12 @@ def register(app, uri, file_or_directory, pattern,
                         del headers['Content-Length']
                         for key, value in _range.headers.items():
                             headers[key] = value
+            headers['Content-Type'] = content_type \
+                or guess_type(file_path)[0] or 'text/plain'
             if request.method == 'HEAD':
                 return HTTPResponse(
                     headers=headers,
-                    content_type=guess_type(file_path)[0] or 'text/plain')
+                    content_type=content_type)
```

Comment:
what happens here if `content_type` is still `None`?

Reply:
Ah, good catch!
Since `headers['Content-Type']` is defined from lines 99-100, `response.HTTPResponse` will default to `headers['Content-Type']` instead of `content_type` regardless if `content_type` is defined:
https://github.com/channelcat/sanic/blob/master/sanic/response.py#L146

Because of the above case, should this line be removed,
or
should this become `content_type=headers['Content-Type']` (feels redundant)?